src/components/CompareResults/ResultsTable.tsx

```diff
@@ -104,21 +104,85 @@ export default function ResultsTable() {
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
   const [frameworkIdVal, setFrameworkIdVal] = useState(frameworkId);
   const [tableFilters, setTableFilters] = useState(
-    new Map() as Map<string, Set<string>>, // ColumnID -> Set<Values to remove>
+    new Map(), // ColumnID -> Set<Values to remove>
   );
+  const filterableKeys = ['status', 'platform', 'confidence'];
 
+  const getPossibleValues = (columnId: string): string[] | undefined => {
+    const config = cellsConfiguration.find(
+      (c) => c.key === columnId,
+    ) as CompareResultsTableConfig;
+    return config.possibleValues;
+  };
   const onClearFilter = (columnId: string) => {
+    const possibleValues = getPossibleValues(columnId);
+
     setTableFilters((oldFilters) => {
       const newFilters = new Map(oldFilters);
       newFilters.delete(columnId);
+      const values = possibleValues?.join(',') || '';
+
+      searchParams.set(columnId, values);
+      setSearchParams(searchParams);
```

Comment:
You need to use the ```useRawSearchParams()``` hook as the ```useSearchParams()``` hook rerenders the page everytime a filter is changed. Look at this pr [here](https://github.com/mozilla/perfcompare/pull/724) to find out more.